src/data.js

```diff
@@ -292,6 +292,31 @@ module.exports = [
       'JavaScript',
     ],
   },
+  {
+    name: 'Shaun Farrell',
+    description:
+      'Full Stack Developer, React, ReactNative, Swift, Model-driven, Node',
+    url: 'https://codedplanet.co/uses',
+    twitter: '@shaunjfaz',
+    emoji: '🙏',
+    country: '🏴󠁧󠁢󠁥󠁮󠁧󠁿',
+    computer: 'apple',
+    phone: 'iphone',
+    tags: [
+      'Developer',
+      'Web Developer',
+      'Full Stack',
+      'Front End',
+      'JavaScript',
+      'TypeScript',
+      'React',
+      'HTML',
+      'CSS',
+      'ReactNative',
```

Comment:
Please put a space in React Native to keep it consistent with other entries - thank you!